.github/workflows/sonar_scan.yml

```diff
@@ -0,0 +1,38 @@
+name: sonar analyzes
```

Comment:
could you please integrate this as a step to existing CI pipelines that are running for PRs and after merging on the main branch ??